src/g3log/filesink.hpp

```diff
@@ -15,11 +15,11 @@ namespace g3 {
 
    class FileSink {
    public:
-      FileSink(const std::string &log_prefix, const std::string &log_directory);
+      FileSink(const std::string &log_prefix, const std::string &log_directory, const std::string &logger_id="g3log");
       virtual ~FileSink();
 
       void fileWrite(LogMessageMover message);
-      std::string changeLogFile(const std::string &directory);
+      std::string changeLogFile(const std::string &directory, const std::string &logger_id="g3log");
```

Comment:
@jkhoogland, the default argument has no impact since when you are calling the ```changeLogFile``` through the sink's handle the argument must be specified (thanks to template issues). 

Please remove the ```="g3log"``` and the PR is good to go